src/UglyToad.PdfPig/Geometry/PdfLine.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace UglyToad.PdfPig.Geometry
+{
+    /// <summary>
+    /// A line in a PDF file. 
+    /// </summary>
+    /// <remarks>
+    /// PDF coordinates are defined with the origin at the lower left (0, 0).
+    /// The Y-axis extends vertically upwards and the X-axis horizontally to the right.
+    /// Unless otherwise specified on a per-page basis, units in PDF space are equivalent to a typographic point (1/72 inch).
+    /// </remarks>
+    public struct PdfLine
+    {
+        /// <summary>
+        /// Length of the line.
+        /// </summary>
+        public decimal Length
+        {
+            get
+            {
+                decimal l = (Point1.X - Point1.X) * (Point1.X - Point1.X) + (Point1.Y - Point1.Y) * (Point1.Y - Point1.Y);
```

Comment:
Does this return the right value? Both the delta X and Y use point1 instead of point1 and point2?